### PR TITLE
Api login logout

### PR DESCRIPTION
**Pull request by pyarati**
flask migration and added formatted readme.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,99 @@
...
+$ db.create_all()
+```
+If we want to add new column in table then we have to drop all table with losing data. 
+
+### Run project
+```
+$ python run.py
+```
+### POSTMAN Collections
+[POSTMAN API](https://www.getpostman.com/collections/73332f547b6b0b6c18d5)
```
Comment (shreyas-damle) [inline README.md]: Instead of putting just link, add the postman collection in the repo.

File: app/__init__.py [hunk 2]
```diff
@@ -3,19 +3,25 @@
...
-app.config['SECRET_KEY'] = '\x1f\x19\xc7\x95\xb6\xac\xd9\x1c\xbd\xd8%V\xd8\x1b@\xdf!\x13A\x9eW8\xa7\xc0'
-app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://root:mySQL@123@localhost/bank_system'
-app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
-app.config["DEBUG"] = True
-app.config["JWT_SECRET_KEY"] = '1234567890abcdefghijklmnopqrstuvwxyz'
-app.config['PROPAGATE_EXCEPTIONS'] = True
+
+app1 = Flask(__name__)
+app1.config['SECRET_KEY'] = '\x1f\x19\xc7\x95\xb6\xac\xd9\x1c\xbd\xd8%V\xd8\x1b@\xdf!\x13A\x9eW8\xa7\xc0'
+app1.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://root:mySQL@123@localhost/bank_system'
```
Comment (shreyas-damle) [inline app/__init__.py]: Use environment variables for username, password and host.

File: migrations/versions/08764d035684_.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
+"""empty message
```
Comment (shreyas-damle) [inline migrations/versions/08764d035684_.py]: I think we do not need these version files in the git repo. Please confirm if they are needed.

Comment (pyarati) [reply migrations/versions/08764d035684_.py]: we do not required this file, I will remove this file.

File: app/__init__.py (hunk 2 above)
Comment (pyarati) [reply app/__init__.py]: Made the changes as suggested.

File: README.md (hunk 1 above)
Comment (pyarati) [reply README.md]: Made the changes as suggested.

**Pull request merged**